bertopic/_bertopic.py

```diff
@@ -1897,9 +1903,40 @@ def get_topic_tree(
 
         max_original_topic = hier_topics.Parent_ID.astype(int).min() - 1
 
+        # Prepare tree labels to print
+        child_left_ids = hier_topics.Child_Left_ID.astype(int)
+        child_right_ids = hier_topics.Child_Right_ID.astype(int)
+
+        # Get the new parent labels generated from `hierarchical_topics`
+        new_left_labels = {int(row["Child_Left_ID"]): row["Child_Left_Name"] for idx, row in hier_topics.iterrows()}
+        new_right_labels = {int(row["Child_Right_ID"]): row["Child_Right_Name"] for idx, row in hier_topics.iterrows()}
+
+        if custom_labels:
+            left_labels = {}
+            if isinstance(custom_labels, str):
+                for topic, kws_info in self.topic_aspects_[custom_labels].items():
+                    label = "_".join([kw[0] for kw in kws_info[:5]])  # displaying top 5 kws
+                    left_labels[topic] = label
+            elif self.custom_labels_ is not None and custom_labels:
+                left_labels = {topic_id: label for topic_id, label in enumerate(self.custom_labels_, -self._outliers)}
```

Comment:
I just wanted to mention that I am going to steal that enumerate trick. That is so much nicer coding than how I have approached it thus far.